components/LoginForm.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import {
+  Box,
+  Text,
+  Heading,
+  VStack,
+  FormControl,
+  Input,
+  Button,
+  Center,
+  Image,
+  Alert,
+  HStack,
+  IconButton,
+  CloseIcon,
+  Collapse,
+  ScrollView,
+} from "native-base";
+import { StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
+  container: {
+    marginTop: 20,
+    textAlign: "center",
+  },
+});
+
+function LoginForm() {
+  const [show, setShow] = React.useState(false);
```

Comment:
For all of these, it's cleaner to import `useState` directly, instead of having all the `React.blabla` everywhere.